packages/riipen-ui/src/components/Breadcrumbs.test.jsx

```diff
@@ -0,0 +1,189 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Breadcrumbs from "./Breadcrumbs.jsx";
+
+describe("<Breadcrumbs>", () => {
+  it("renders without errors", () => {
+    let error;
+    const child = <h1>hello hello</h1>;
+    try {
```

Comment:
Following you conventions down below, it seems like you are always doing

```js
[arrange]
_blank line_
[act]
_blank line_
[assert]
```

Following that pattern, I think there should be a blank line between 10 and 11 as that's the gap between your arrange and assert here.